src/dso_api/dynamic_api/views/base.py

```diff
@@ -0,0 +1,143 @@
+from collections.abc import Generator
+from itertools import chain
+
+from django.contrib.gis.db.models.functions import Transform
+from django.db import connections
+from django.http import HttpResponse, StreamingHttpResponse
+from django.views import View
+from vectortiles.backends import BaseVectorLayerMixin
+from vectortiles.backends.postgis.functions import AsMVTGeom, MakeEnvelope
+from vectortiles.mixins import BaseVectorTileView
+
+
+class StreamingVectorLayer(BaseVectorLayerMixin):
```

Comment:
Aangezien deze base logica alleen mvt betreft zou je dit eventueel in `mvt.py` kunnen zetten? of een `mvt_base.py` kunnen aanhouden, om toch die library code iets apart te houden van onze logica.